### DISCUSSION TITLE
Error installing Wireguard, [ERROR] in line 26: exit code 0

### DISCUSSION THREAD

**Discussion by Jorgeelcid** [tteck/Proxmox#3644] [Help Needed] [answered]
I'm trying to install WireGuard, but when the script runs line 26 throws this error: 

[ERROR] in line 26: exit code 0: while executing command DEBIAN_FRONTEND=noninteractive apt-get -o Dpkg::Options::="--force-confnew" install -y iptables-persistent &> /dev/null

I'm using pve shell with root.

**Comment by tteck** [comment, marked as answer]
Fixed
